src/js/components/componentDiagram/graph/util.js

```diff
@@ -0,0 +1,97 @@
+export function generateHash() {
+  return Math.random().toString(16).slice(2, 10);
+}
+
+export function getAnimationStep(from, to, percent) {
+  return percent < 1 ? from + (to - from) * percent : to;
+}
+
+export function createSVGElement(tagName, className = null) {
+  const el = document.createElementNS('http://www.w3.org/2000/svg', tagName);
+
+  if (className) {
+    el.setAttribute('class', className);
+  }
+
+  return el;
+}
+
+export function transformPoints(points) {
```

Comment:
Can you add a comment to explain what this is?

Reply:
Added comment: `transform 'dagre' points into simple array which will be passed to 'd3-line' helper`